packages/amplify_datastore/ios/Classes/SwiftAmplifyDataStorePlugin.swift

```diff
@@ -8,16 +8,19 @@ import AmplifyPlugins
 import AWSPluginsCore
 import AWSCore
 import Combine
-import amplify_flutter_ios
 
-public class SwiftAmplifyDataStorePlugin: NSObject, FlutterPlugin {
+public class SwiftAmplifyDataStorePlugin: NSObject, FlutterPlugin, NativeAmplifyBridge, NativeAuthBridge, NativeApiBridge {
     private let bridge: DataStoreBridge
     private let modelSchemaRegistry: FlutterSchemaRegistry
     private let customTypeSchemaRegistry: FlutterSchemaRegistry
     private let dataStoreObserveEventStreamHandler: DataStoreObserveEventStreamHandler?
     private let dataStoreHubEventStreamHandler: DataStoreHubEventStreamHandler?
     private var channel: FlutterMethodChannel?
     private var observeSubscription: AnyCancellable?
+    private static var nativeAuthPlugin: NativeAuthPlugin?
+    static var nativeApiPlugin: NativeApiPlugin?
+    private var cognitoPlugin: CognitoPlugin?
```

Comment:
I would make private instance vars and initialize in `init`. For `FlutterAuthProviders`, we can pass in the instance var instead.

```suggestion
    private let nativeAuthPlugin: NativeAuthPlugin
    private let nativeApiPlugin: NativeApiPlugin
    private let cognitoPlugin: CognitoPlugin
```

Reply:
Updated